tests/test_mpi.py

```diff
@@ -1005,6 +1008,183 @@ def test_avoid_haloupdate_if_distr_but_sequential(self, mode):
         calls = FindNodes(Call).visit(op)
         assert len(calls) == 0
 
+    @pytest.fixture
+    def setup(self):
+        shape = (2,)
+        so = 2
+        tn = 30
+
+        grid = Grid(shape=shape)
+
+        # Velocity and pressure fields
+        v = TimeFunction(name='v', grid=grid, space_order=so)
+        tau = TimeFunction(name='tau', grid=grid, space_order=so)
+
+        # First order elastic-like dependencies equations
+        pde_v = v.dt - (tau.dx)
+        pde_tau = tau.dt - ((v.forward).dx)
+        u_v = Eq(v.forward, solve(pde_v, v.forward))
+        u_tau = Eq(tau.forward, solve(pde_tau, tau.forward))
+
+        # Receiver
+        rec = SparseTimeFunction(name="rec", grid=grid, npoint=1, nt=tn)
+        rec.coordinates.data[:, 0] = np.linspace(0., shape[0], num=1)
+
+        return grid, v, tau, u_v, u_tau, rec
+
+    @pytest.mark.parallel(mode=1)
+    def test_issue_2448_I(self, mode, setup):
```

Comment:
nitpicking: we use v0, v1, v2 ... not roman numbers (again, nothing against it, but we need code homogeneity)